packages/data-format/src/index.ts

```diff
@@ -1,4 +1,5 @@
-import * as AJV from 'ajv';
+import { Ajv } from 'ajv';
```

Comment:
There was an issue with importing Ajv with the old version. I fixed it by updating the package. There was some breaking changes.

Later on, I turned `esModuleInterop=true` in the `tsconfig.json`, which may also have resolved this issue without updating the `Ajv` package. I haven't rolled back the Ajv update though...